ovirt-ansible-collection.spec.in

```diff
@@ -12,8 +12,11 @@ Group:          Virtualization/Management
 BuildArch:      noarch
 Url:            http://www.ovirt.org
 
-Requires: ansible >= 2.9.0
+Requires: ansible >= 2.9.11
```

Comment:
What is the reason for this change?
If it is required by hosted engine update also readme that we require ansible "2.9.11 or higher"

Reply:
Ansible changed the behavior in 2.9.11 and HE deployment failed (https://bugzilla.redhat.com/show_bug.cgi?id=1859483)
I'll update the README.